department-of-veterans-affairs/caseflow
# Building organization frontend

Connects #6392 

### Description
Created a new page at /organizations/:organization_name. This new page is just a list of appeals with tasks assigned to the organization.

### Acceptance Criteria 
- [ ] `/organizations/:organization_name` has a list of all the appeals with tasks assigned to the organization.

### Testing Plan
1. Run `rake db:seed`
1. Change to the `VSO` user
1. Go to `/organizations/american-legion`
1. Ensure there is an appeal
